Ombi.Api/TheMovieDbApi.cs

```diff
@@ -100,5 +105,48 @@ public TmdbMovieDetails GetMovieInformationWithVideos(int tmdbId)
             var movies = await Client.GetMovie(imdbId);
             return movies ?? new Movie();
         }
+
+        public async Task<List<Movie>> SearchActor(string searchTerm)
+        {
+            return await SearchActor(searchTerm, null);
+        }
+
+        public async Task<List<Movie>> SearchActor(string searchTerm, Func<int, string, string, Task<bool>> alreadyAvailable)
+        {
+            SearchContainer<SearchPerson> result = await Client.SearchPerson(searchTerm);
+            
+            var people = result?.Results ?? new List<SearchPerson>();
+            var person = (people.Count != 0 ? people[0] : null);
+            var movies = new List<Movie>();
+            var counter = 0;
+            try
+            {
+                if (person != null)
+                {
+                    var credits = await Client.GetPersonMovieCredits(person.Id);
+                    //only get the first 10 movies and delay a bit between each request so we don't overload the API
+                    foreach (var credit in credits.Cast)
+                    {   if (counter == 10)
+                            break;
+                        if (alreadyAvailable == null || !(await alreadyAvailable(credit.Id, credit.Title, credit.ReleaseDate.Value.Year.ToString())))
+                        {
+                            movies.Add(await GetMovie(credit.Id));
+                            counter++;
+                        }
+                        else
+                        {
+                            //if we didn't add the movie, delay longer since we know we'll be making additional API calls
+                            await Task.Delay(75);
```

Comment:
Given the API key/rate information, I'm going to rip out the extra 75ms.  It would actually be neat to build in some throttling at the TMDbAPI level. How active is that project?